### PR TITLE
AWS release 9.3.7

### PR DESCRIPTION
**Pull request by calvix**
release to fix bug https://github.com/giantswarm/giantswarm/issues/12450

### CODE REVIEW

File: aws/v9.3.7/release.diff [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  - name: cluster-operator                                           - name: cluster-operator
+    releaseOperatorDeploy: true                                        releaseOperatorDeploy: true
+    version: 0.23.14                                                   version: 0.23.14
+  - name: containerlinux                                             - name: containerlinux
+    version: 2512.2.1                                                  version: 2512.2.1
+  - name: etcd                                                       - name: etcd
+    version: 3.3.17                                                    version: 3.3.17
+  - name: kubernetes                                                 - name: kubernetes
+    version: 1.16.12                                                   version: 1.16.12
+  date: 2020-07-31T15:00:00Z                                    |    date: "2020-08-19T15:39:57Z"
```
Comment (njuettner) [inline aws/v9.3.7/release.diff:53]: bump the date 🙂

File: aws/v9.3.7/release.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  - name: cluster-operator
+    releaseOperatorDeploy: true
+    version: 0.23.14
+  - name: containerlinux
+    version: 2512.2.1
+  - name: etcd
+    version: 3.3.17
+  - name: kubernetes
+    version: 1.16.12
+  date: "2020-08-19T15:39:57Z"
```
Comment (njuettner) [inline aws/v9.3.7/release.yaml:53]: also here

**Review by njuettner — APPROVED** [review]
lgtm

**Comment by calvix** [comment]
cluster creation `v9.3.7` and update from `v9.3.6` to `v9.3.7`

**Pull request merged**